docs/src/main/sphinx/sql/deny.rst

```diff
@@ -0,0 +1,51 @@
+====
+DENY
+====
+
+Synopsis
+--------
+
+.. code-block:: text
+
+    DENY ( privilege [, ...] | ( ALL PRIVILEGES ) )
+    ON ( table_name | TABLE table_name | SCHEMA schema_name)
+    TO ( user | USER user | ROLE role )
+
+Description
+-----------
+
+Denies the specified privileges to the specified grantee.
+
+Specifying ``ALL PRIVILEGES`` denies :doc:`delete`, :doc:`insert`, :doc:`update` and :doc:`select` privileges.
+
+Specifying ``ROLE PUBLIC`` denies privileges to the ``PUBLIC`` role and hence to all users.
+
+Deny on a table rejects the specified privilege on all current and future columns of the table.
+
+Deny on a schema rejects the specified privilege on all current and future columns of all current and future tables of the schema.
+
+Examples
+--------
+
+Deny ``INSERT`` and ``SELECT`` privileges on the table ``orders`` to user ``alice``::
+
+    DENY INSERT, SELECT ON orders TO alice;
+
+Deny ``DELETE`` privilege on the schema ``finance`` to user ``bob``::
+
+    DENY DELETE ON SCHEMA finance TO bob;
+
+Deny ``SELECT`` privilege on the table ``orders`` to everyone::
+
+    DENY SELECT ON orders TO ROLE PUBLIC;
+
+Limitations
+-----------
+
+Some connectors have no support for ``DENY``.
+See connector documentation for more details.
```

Comment:
This is also a feature of system access controls, and I expect that it will be used in those more than in connector access controls.

Reply:
I modified the sentence by mentioning system access controls as well.